docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/tracks.md

```diff
@@ -13,9 +13,39 @@ status: draft
 
 # Tracks in the Genome browser
 
-Tracks are sets of data that can be plotted against the genome, such as transcripts, genetic variation or conserved regions. Entities within tracks are shown on horizontal rows. There may be multiple rows for a track, ensuring that there is no overlap within a row. Some tracks, such as genes, are associated with one strand of the genome – behind stranded tracks you will see small arrows pointing in the direction of transcription.
+## What is a track ? 
 
-You can modify the appearance of tracks by clicking on the cog icon to the right of the track. This allows you to show or hide labels on the track and/or the entity, to change the height of the track, to highlight it, to move it and to lock it.
+Tracks are sets of data such as genes, variants or regulatory motifs displayed along a genome. A track displays a single set of data ie genes on the forward strand.
```

Comment:
Regulatory features is a better term than motifs (the latter has a meaning of its own and is not what's shown). Fine to include it I think even though it's not present yet.

Could also say 'eg protein coding genes on the forward strand' since that is a track we have

Reply:
Done